src/components/shared/Loader.tsx

```diff
@@ -0,0 +1,16 @@
+export default function Loader (): JSX.Element {
```

Comment:
```suggestion
export function Loader (): JSX.Element {
```

don't use default export, they make refactoring a pain in the ass

I will enforce these lint rules later, but you make the change first